lib/request-wrapper.ts

```diff
@@ -67,42 +68,42 @@ export class RequestWrapper {
     this.axiosInstance = axios.create(axiosConfig);
 
     // set debug interceptors
-    if(process.env.NODE_DEBUG === 'axios') {
+    if(process.env.NODE_DEBUG === 'axios' || process.env.DEBUG) {
       this.axiosInstance.interceptors.request.use(config => {
-        console.debug('Request:');
+        logger.debug('Request:');
         try {
-          console.debug(JSON.stringify(config, null, 2));
+          logger.debug(JSON.stringify(config, null, 2));
         } catch {
-          console.debug(config)
+          logger.error(config)
         }
 
         return config;
       }, error => {
-        console.debug('Error:');
+        logger.error('Error sending request: ');
         try {
-          console.debug(JSON.stringify(error, null, 2));
+          logger.error(JSON.stringify(error, null, 2));
         } catch {
-          console.debug(error);
+          logger.error(error);
         }
 
         return Promise.reject(error);
       });
 
       this.axiosInstance.interceptors.response.use(response => {
-        console.debug('Response:');
+        logger.debug('Response:');
         try {
-          console.debug(JSON.stringify(response, null, 2));
+          logger.debug(JSON.stringify(response, null, 2));
         } catch {
-          console.debug(response)
+          logger.error(response);
         }
 
         return response;
       }, error => {
-        console.debug('Error:');
+        logger.error('Error returning response: ');
         try {
-          console.debug(JSON.stringify(error, null, 2));
+          logger.error(JSON.stringify(error, null, 2));
```

Comment:
If the error logged here does not include the original request, it would be really nice if we could find a way to log the request here as well.

Reply:
So I ran a test and it looks like the error object returned does in fact contain the original request (under ``error.config``), so as long as the stringify has been doing its job then that should have been logged as well